xbmc/music/MusicDatabase.cpp

```diff
@@ -4576,33 +4644,22 @@ bool CMusicDatabase::GetAlbumsByWhere(const std::string &baseDir, const Filter &
     }
     querytime = XbmcThreads::SystemClockMillis() - querytime;
 
-    // store the total value of items as a property
+    // Store the total number of albums as a property
     if (total < iRowsFound)
       total = iRowsFound;
     items.SetProperty("total", total);
-
-    if (countOnly)
-    {
-      CFileItemPtr pItem(new CFileItem());
-      pItem->SetProperty("total", total);
-      items.Add(pItem);
-
-      m_pDS->close();
-      return true;
-    }
-
+   
```

Comment:
remove the white space at the end of the line

Reply:
whitespace fixed

Reply:
and it's back again

Reply:
Do you know what, I really don't care about a white space. I think it is trivial beyond belief.
Meanwhile an actual review of functionality and design could be worthwhile, oh well.